packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -8,6 +8,11 @@ import {ResultUtils} from 'c/quanticUtils';
 import {NavigationMixin} from 'lightning/navigation';
 import {LightningElement, api} from 'lwc';
 
+/**
+ * Some document types cannot be opened directly in Salesforce, but we need to open their parent record, as is the case for the Case Comment document type.
+ */
+const documentTypesRequiringParentRecord = ['CaseComment'];
```

Comment:
Have you tried if it's possible to have the comment of a comment?

Aka try to figure out in which situation this is **not** actually working.

I don't exactly know anymore, but there's two types of objects that can be considered "CaseComments" too, the first one is an old feature called Case Comment, and the 2nd one are what we call specific FeedItems records where the parent is a Case.

But there's also FeedComment where the parent can be a FeedItem on a case, etc.

all in all make sure it works with grand-children too etc.

Reply:
The logic I added was meant to be easily scalable to adapt to any other similar situation/issue that we would face. 
But for now that's the only case where I managed to reproduce the issue with.